### PR TITLE
Moving logic assets to main game

### PR DESCRIPTION
**Pull request by KingJeremyNg**

### CODE REVIEW

**Comment by KingJeremyNg** [comment]
Initial pull request to get logic team started

File: src/routes.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import Home from './routes/home.svelte'
 import Pacman from './routes/pacman/pacman.svelte'
+import TowerDefense from './routes/td/td.svelte'
```
Comment (Squishy123) [inline src/routes.js]: Change folder names
```suggestion
import TowerDefense from './routes/towerdefense/towerdefense.svelte'
```

File: src/routes/td/classes/player.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+export default class Player {
```
Comment (Squishy123) [inline src/routes/td/classes/player.js]: These are probably gonna be actors?

File: src/routes/td/td.svelte [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<!--<script>
```
Comment (Squishy123) [inline src/routes/td/td.svelte]: Set this up. Look at pacman demo

**Review by Squishy123 — CHANGES_REQUESTED** [review]

**Review by Squishy123 — APPROVED** [review]
Looks good to me!

**Pull request merged**